src/components/UNSTABLE_Pagination/Pagination.js

```diff
@@ -0,0 +1,248 @@
+/**
+ * @file Unstable pagination.
+ * @copyright IBM Security 2019
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { CaretRight24, CaretLeft24 } from '@carbon/icons-react';
+import { Button, Select, SelectItem } from 'carbon-components-react';
+import { getComponentNamespace } from '../../globals/namespace';
+
+export const namespace = getComponentNamespace('unstable-pagination');
+
+function UNSTABLE_Pagination({
+  backwardText,
+  children,
+  className,
+  disabled,
+  forwardText,
+  id,
+  itemsPerPageText,
+  itemRangeText,
+  itemText,
+  page,
+  pageRangeText,
+  pageSize,
+  pageSizes,
+  pageText,
+  pagesUnknown,
+  totalItems,
+  ...rest
+}) {
+  const [currentPage, setCurrentPage] = useState(page);
+  const [currentPageSize, setCurrentPageSize] = useState(pageSize);
+
+  const totalPages = Math.max(Math.ceil(totalItems / currentPageSize), 1);
+
+  const backButtonDisabled = disabled || currentPage === 1;
+  const forwardButtonDisabled = disabled || currentPage === totalPages;
+
+  function onSetPage(newPage) {
+    setCurrentPage(Number(newPage));
+  }
+
+  return (
```

Comment:
Similar to a comment in `_mixins.scss`, is there an opportunity for us to reuse the styles from Carbon?